namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-dev/resources/rds.tf

```diff
@@ -37,7 +38,7 @@ resource "kubernetes_secret" "court_case_service_rds" {
     url                        = "postgres://${module.court_case_service_rds.database_username}:${module.court_case_service_rds.database_password}@${module.court_case_service_rds.rds_instance_endpoint}/${module.court_case_service_rds.database_name}"
     access_key_id              = module.court_case_service_rds.access_key_id
     secret_access_key          = module.court_case_service_rds.secret_access_key
-    db_backup_retention_period = 0
+    db_backup_retention_period = module.court_case_service_rds.db_backup_retention_period
```

Comment:
@chris-faulkner-mojdt , this will not set the value as the RDS module currently doesnot return the db_backup_retention_period value. In order to see the changes, you have to check via the CLI. `aws rds describe-db-instances --db-instance-identifier <mydbinstance>`